src/Akka.Persistence.Linq2Db.Data.Compatibility.Tests/DataCompatibilitySpecBase.cs

```diff
@@ -134,35 +139,21 @@ public async Task DisposeAsync()
         protected async Task TruncateEventsToLastSnapshot()
         {
             var region = TestCluster!.ShardRegions[0];
-            var cts = new CancellationTokenSource(10.Seconds());
-            try
+            using var cts = new CancellationTokenSource(10.Seconds());
+            foreach (var id in Enumerable.Range(0, 100))
```

Comment:
* Use `using` instead of `try...finally`
* Delete single event at a time instead of batching all deletes, problematic with MySql and PostgreSql connection pool